src/promptflow/tests/sdk_cli_azure_test/conftest.py

```diff
@@ -271,6 +272,63 @@ def created_flow(pf: PFClient, randstr: Callable[[str], str]) -> Flow:
     yield result
 
 
+@pytest.fixture
+def created_batch_run_without_llm(pf: PFClient, randstr: Callable[[str], str], runtime: str) -> Run:
+    """Create a batch run that does not require LLM."""
+    name = randstr("batch_run_name")
```

Comment:
not sure if re-entrance of this randstr will just create a same name or a different name

Reply:
if in same session, the name will keep same, as the inner implementation is a get or set operation; otherwise, the value will be different

Reply:
ignore me